test/base_test.py

```diff
@@ -101,6 +101,34 @@ def test_data_on_slice(self, server, format_arg, flatten):
 
         assert retrieved_data - data[100, 0] < 1e-8
 
+    def test_meta_on_chunked_compressed_dataset(self, server):
+        """Test /meta/ endpoint on a chunked and compressed dataset"""
+        filename = "test.h5"
+        tested_h5entity_path = "/data"
+
+        with h5py.File(server.served_directory / filename, mode="w") as h5file:
+            h5file.create_dataset(
+                tested_h5entity_path,
+                data=np.arange(100).reshape(10, 10),
+                compression="gzip",
+                shuffle=True,
+                dtype="<f8",
+                chunks=(5, 5),
+            )
+
+        response = server.get(f"/meta/?file={filename}&path={tested_h5entity_path}")
+        content = decode_response(response)
+
+        assert content == {
+            "attributes": [],
+            "compression": [{"id": 2, "name": "shuffle"}, {"id": 1, "name": "deflate"}],
```

Comment:
I'd keep both at least in the backend.
The unique identifier is the `id`, the `name` is a random string provided by the filter... it might be neither meaningful nor unique.